readme.md

```diff
@@ -71,6 +77,13 @@ Default: `false`
 
 Uppercase the first character: `foo-bar` → `FooBar`
 
+##### locale
+
+Type: `string | string[]`\
+Default: `undefined`
+
+Locale to be used with lowercase or uppercase conversions.
```

Comment:
This needs to explicitly say what the expected string locale format it. (for example, what `en-US` refers to).

Reply:
It's also unclear why one would want to specify multiple locales and how that works.

Reply:
I updated with an explicit reference to [String.prototype.toLocaleUpperCase()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLocaleUpperCase) and excerpt from that page.  It explains format and usage with multiple locale.
Details (like expected format being [BCP 47](https://tools.ietf.org/html/bcp47) etc.) seemed too much in this context, but it  is up to you.

Reply:
One example in Usage and all in the API docs for `locale`.

Reply:
done.